lib/matplotlib/text.py

```diff
@@ -1838,9 +1838,12 @@ def transform(renderer) -> Transform
                 self._arrow_relpos = arrowprops.pop("relpos", (0.5, 0.5))
             else:
                 # modified YAArrow API to be used with FancyArrowPatch
-                for key in [
-                        'width', 'headwidth', 'headlength', 'shrink', 'frac']:
+                for key in ['width', 'headwidth', 'headlength', 'shrink']:
                     arrowprops.pop(key, None)
+                if 'frac' in key:
```

Comment:
```suggestion
                if 'frac' in arrowprops:
```

`key` will be "shrink" from the for loop

Reply:
woops, fixed